js/functional/palindrome.js

```diff
@@ -1,13 +1,25 @@
 import { parseString } from "../helper.js";
 
+/**
+ * Add cleanString to avoid repeating the cleaning process for cleanedValue in each function.
+ *
+ * @param {string} value
+ */
+function cleanString(value) {
+  return value.toLowerCase().replace(/[^a-z0-9]/g, "");
+}
```

Comment:
This function could be placed at "js/helper.js" as it can be used by any function besides palindrome.